src/main/java/hudson/remoting/Request.java

```diff
@@ -79,7 +79,7 @@
      */
     private int lastIoId;
 
-    private volatile Response<RSP,EXC> response;
+    private volatile Response<RSP,? extends Throwable> response;
```

Comment:
The original type parameter was wrong. Fortunately, this never resulted in any actual bugs due to type erasure. The reasoning is as follows. The `EXC` type parameter in the `Request` class represents the bound for the exception types that subclasses of `Request` may produce. The `Response` class may include exceptions of these types, but it may also include an exception of type `RequestAbortedException` (see `Request#abort(IOException)`). Recall that this field is in the `Request` class, so `EXC` represents the bound on the _request_'s exception types, not the _response_'s exception types. To keep supporting `Request#abort(IOException)`, the bound here must be able to include `RequestAbortedException`. So the bound for `EXC` is not large enough, so we must include all instances of `Throwable`. This also prevents us from having to make a risky change to the `Runnable` in `Request#execute`.